File: 0010-0019/0016.rb
gx, gy = 0, 0
th = 90
while gets do
  st, rot = $_.split(',').map(&:to_i)
  break if st == 0 && rot == 0
  gx = gx + st*Math.cos(th/180.0*Math::PI)
  gy = gy + st*Math.sin(th/180.0*Math::PI)
  th -= rot
end
puts [gx.to_i, gy.to_i]

File: python/0016.py
import sys
import math

gx = gy = 0.0
th = 90.0
for s in sys.stdin:
  x, rot = map(int, s.split())
  gx += x*math.cos(th/180*math.pi)
  gy += x*math.sin(th/180*math.pi)
  th -= rot

print int(gx)
print int(gy)